platform/src/main/java/org/stellar/anchor/platform/rpc/NotifyRefundPendingHandler.java

```diff
@@ -185,6 +188,10 @@ protected void updateTransactionWithRpcRequest(
       case SEP_6:
         JdbcSep6Transaction txn6 = (JdbcSep6Transaction) txn;
 
+        if (ImmutableSet.of(WITHDRAWAL, WITHDRAWAL_EXCHANGE).contains(Kind.from(txn6.getKind()))) {
```

Comment:
Can we add a comment here explaining why we don't update the transaction for withdrawals?